src/App.js

```diff
@@ -64,23 +66,23 @@ export default function App() {
               pathNameRedirect="/"
               isLoggedIn={loggedIn}
             >
-              <OrgTopLevel authObject={authObject} />
+              <OrgTopLevel />
             </PrivateRoute>
 
             <PrivateRoute
               path="/generateLabel"
               pathNameRedirect="/"
               isLoggedIn={loggedIn}
             >
-              <PdfGenerator authObject={authObject} />
+              <PdfGenerator />
             </PrivateRoute>
 
             <PrivateRoute
```

Comment:
I implemented the context, but I personally prefer passing the components as a child, I think it makes the routing easier to understand, rather than a bunch of nearly-identical components